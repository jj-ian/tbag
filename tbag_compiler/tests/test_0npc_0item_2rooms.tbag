/* Authors: Julie */
room {
	int temperature;
}

room Space {
	name = "space";
	temperature = neg 1000;
}

room Sun {
	name = "surface of the sun";
	temperature = 99999;
}

Space <-> Sun;

start { Space }

currentRoom == Space {
	print("you are in ");
	print(currentRoom.name);
	print(". you're being launched into the surface of the sun\n");
	print("currently the temperature is ");
	print(currentRoom.temperature);
	print("\n");
	->Sun
}

currentRoom == Sun {
	print("Now you're on the ");
	print(currentRoom.name);
	print(" the temp is ");
	print(currentRoom.temperature);
	endgame;
}
