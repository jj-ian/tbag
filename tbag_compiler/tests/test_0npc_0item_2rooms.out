you are in space. you're being launched into the surface of the sun
currently the temperature is -1000
Now you're on the surface of the sun the temp is 99999